table/index.go

```diff
@@ -33,12 +33,25 @@ type CreateIdxOpt struct {
 	SkipCheck       bool // If true, skip all the unique indices constraint check.
 }
 
+// CreateIdxOption is defined for the Create() method of Index interface.
+type CreateIdxOption func(*CreateIdxOpt)
```

Comment:
How about `CreateIdxOptFunc`?
It's hard to distinguish any difference between `CreateIdxOption` and `CreateIdxOpt `.

Reply:
Done